t/redirect.t

```diff
@@ -0,0 +1,41 @@
+use strict;
+use warnings;
+use Test::More;
+
+use LWP::UserAgent;
+plan tests => 4;
+
+# This is a regression test for #171
+
+my $ua = LWP::UserAgent->new();
+
+{ # default number of redirects
+    $ua->timeout(1);
+    my $res = $ua->get('http://10.255.255.1/');
```

Comment:
`10.255.255.1` is a private address. Some routers use this as their default setup address. This could actually respond on someone's home network.

Is there a better address to use for this purpose?